digital-workspace/4.3/develop/manage.md

```diff
@@ -0,0 +1,167 @@
+    ```
+
+    > **Note:** The Digital Workspace 4.3.x uses the Content Application version 4.0.0.
+
+2. Run the following commands to start the application:
+
+    ```bash
+    npm ci
+    npm start content-ee
+    ```
+
+    The application is available at `http://localhost:4200`. Use the Content Services credentials to log in.
+
+    The configuration provided for the sample above is the minimal required for your development environment to locally test the Alfresco Repository, but additional environment variables are available.
+
+    ```bash
+    # App config settings
+    APP_CONFIG_BPM_HOST="<url>"
+    APP_CONFIG_ECM_HOST="<url>"
+    APP_CONFIG_OAUTH2_HOST="<url>"
+    APP_CONFIG_IDENTITY_HOST="<url>"
+    APP_CONFIG_PROVIDER="ALL"
+    APP_CONFIG_AUTH_TYPE="OAUTH"
+    APP_CONFIG_OAUTH2_CLIENTID="alfresco"
+    APP_CONFIG_OAUTH2_IMPLICIT_FLOW=true
+    APP_CONFIG_OAUTH2_SILENT_LOGIN=true
+    APP_CONFIG_OAUTH2_REDIRECT_SILENT_IFRAME_URI="{protocol}//{hostname}{:port}/assets/silent-refresh.html"
+    APP_CONFIG_OAUTH2_REDIRECT_LOGIN=/
+    APP_CONFIG_OAUTH2_REDIRECT_LOGOUT=/
+    APP_CONFIG_APPS_DEPLOYED="[{"name": "simpleapp"}]"
+    APP_CONFIG_LANDING_PAGE="/personal-files"
+
+    # CONTENT RELATED
+    ACA_BRANCH="develop"
+    APP_CONFIG_PLUGIN_PROCESS_SERVICE=true
+    APP_CONFIG_PLUGIN_AI_SERVICE=true
+    APP_CONFIG_PLUGIN_AOS=true
+    APP_CONFIG_PLUGIN_CONTENT_SERVICE=true
+    APP_CONFIG_CUSTOM_MODELED_EXTENSION = "{}"
+
+    # CONTENT - MICROSOFT PLUGIN RELATED
+    APP_CONFIG_PLUGIN_MICROSOFT_ONLINE=true
+    APP_CONFIG_MICROSOFT_ONLINE_OOI_URL="<url>"
+    APP_CONFIG_MICROSOFT_ONLINE_CLIENTID="<clientid>"
+    APP_CONFIG_MICROSOFT_ONLINE_AUTHORITY="<url>"
+    APP_CONFIG_MICROSOFT_ONLINE_REDIRECT="<url>"
+
+    # CONTENT - MICROSOFT INTEGRATION TEST RELATED
+    MICROSOFT_USER_INITIALS="<user-initials>"
+    MICROSOFT_EMAIL="<email>"
+    MICROSOFT_USER2_INITIALS="<user-initials>"
+    MICROSOFT_EMAIL2="<email>"
+    MICROSOFT_PASSWORD="<password>"
+    MOO_LOGIN_URL="<url>"
+    ```
+
+## Build, promote, test
+
+The Digital Workspace is a standard Angular application, and its lifecycle follows the same principles and best practices of any other standard Angular application. Use this information to learn how to install it using different distributions, how to add or remove unwanted modules, how to promote the application for use in different environments, and how to test it.
+
+1. In a command prompt, enter:
+
+    ```bash
+    npm install
+    ```
+
+    There are two distributions available to run with the Digital Workspace:
+
+    | Distribution | Description |
+    | ------------ | ----------- |
+    | `content-ee` | *Default.* The Digital Workspace with the Process Services extension |
+    | `content-ee-apa` | The Digital Workspace with the Process Automation extension |
+
+2. Select the command to start the application:
+
+    ```bash
+    npm start <content-ee|content-ee-apa> [prod]
```

Comment:
`npm start content-ee-apa` won't work with OOTB ADW, at least we don't have it right now in our repo

Reply:
`npm start content-ee` for plain ADW without APA